replays/search.php

```diff
@@ -92,12 +92,12 @@
 		// no
 	} else if ($username) {
 		if (!$isPrivate || $isPrivateAllowed) {
-			$replays = $Replays->search($username, $page, $isPrivate);
+			$replays = $Replays->search(["p1" => $username, "isPrivate" => $isPrivate, "page" => $page]);
```

Comment:
On second thought, I don't know if I like `p1` instead of just `userid` to mean "either p1 or p2". Let's go with `userid`?

Reply:
I chose `p1` for symmetry. Also, `p1` vs. `p2` is a mater of perspective...

I'd prefer `player` and `opponent` to `userid` and `userid2` or `p2` or whatever else, but `p1` and `p2` is most concise.